src/App.tsx

```diff
@@ -98,15 +95,16 @@ export class App extends React.Component<{}, State> {
 
     const visibleGoods = getReorderedGoods(goodsFromServer, this.state);
 
+    const resetCheck = isReversed || sortType !== SortType.NONE;
```

Comment:
> `resetCheck`

is this a function?

Reply:
Hello @naziamoff, this is a variable that store multiple conditions for the rest, the first name that came to my mind :)